tests/ios/xcodeproj/build.sh

```diff
@@ -6,9 +6,8 @@ cd $(dirname $0)
 xcrun simctl list
 
 export SAMPLE_PROJECT_AND_SCHEME="-project Single-Static-Framework-Project.xcodeproj -scheme ObjcFrameworkTests"
-export SIM_DEVICE_ID=`xcodebuild $SAMPLE_PROJECT_AND_SCHEME -showdestinations | grep "platform:iOS Sim" | head -1 | ruby -e "puts STDIN.read.split(',')[1].split(':').last"`
+export SIM_DEVICE_ID=$(xcodebuild $SAMPLE_PROJECT_AND_SCHEME -showdestinations -destination "generic/platform=iOS Simulator" | grep "platform:iOS Sim" | head -1 | ruby -e "puts STDIN.read.split(',')[1].split(':').last")
```

Comment:
Do you still need to do `grep "platform:iOS Sim"` ?

Reply:
yes